tests/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def get_resource_path(filename):
```

Comment:
Oh, interesting -- do we have a usage example elsewhere in the repo? I went with this since this appears to be the current pattern in other tests, but it's repeated in a few places (the intention being that we'd DRY the other instances up over time.)